catalog/imaging/pom.xml

```diff
@@ -28,17 +28,23 @@
     <name>Alliance :: Imaging</name>
 
     <properties>
-        <nitf-imaging.version>0.7</nitf-imaging.version>
+        <nitf-imaging.version>0.8-SNAPSHOT</nitf-imaging.version>
         <camel.version>2.16.1</camel.version>
         <jai-imageio-core.version>1.3.1</jai-imageio-core.version>
         <jpeg2000.version>1.3.1_CODICE_2</jpeg2000.version>
+        <usng4j.version>0.1</usng4j.version>
```

Comment:
I think it's better to keep the properties scoped as privately as we can without duplicating them.  As far as I know the usng packages aren't used outside of the imaging-app bundles, so I'd like to leave this here.  I am removing the duplicate one.